package.json

```diff
@@ -19,19 +19,18 @@
     "@rollup/plugin-multi-entry": "^6.0.1",
     "@rollup/plugin-node-resolve": "^15.3.0",
     "@types/chrome": "^0.0.277",
-    "@typescript-eslint/eslint-plugin": "^7.4.0",
-    "@typescript-eslint/parser": "^7.4.0",
     "babel-plugin-istanbul": "^7.0.0",
     "babel-plugin-mockable-imports": "^2.0.1",
     "chai": "^5.1.1",
     "chrome-webstore-upload-cli": "^3.3.1",
     "diff": "^7.0.0",
-    "eslint": "^8.57.0",
-    "eslint-config-hypothesis": "^2.6.0",
-    "eslint-plugin-mocha": "^10.4.1",
-    "eslint-plugin-react": "^7.34.1",
-    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint": "^9.12.0",
+    "eslint-config-hypothesis": "^3.0.0",
+    "eslint-plugin-mocha": "^10.5.0",
+    "eslint-plugin-react": "^7.37.1",
+    "eslint-plugin-react-hooks": "^5.0.0",
```

Comment:
This project does not really need these two plugins, but they are `eslint-config-hypothesis` peer dependencies.

Reply:
We could add separate entry points in future for Preact and non-Preact using libs/apps.